galley/pkg/server/settings/args.go

```diff
@@ -51,6 +53,12 @@ type Args struct { // nolint:maligned
 	// KubeInterface has an already created K8S interface, will be reused instead of creating a new one
 	KubeInterface *kubernetes.Clientset
 
+	// InsecureGRPC is an existing GRPC server, will be used by Galley instead of creating its own
+	InsecureGRPC *grpc.Server
+
+	// SecureGRPC is an existing GRPC server, will be used by Galley instead of creating its own
+	SecureGRPC *grpc.Server
```

Comment:
I thought istiod had a sidecar?

Reply:
No - based on feedback from TOC, when we discussed it. 
Also given the fact we use k8s-signed certs, we may want to start the https/grpc using the in-memory certificates without the extra copy and complexity.

Reply:
We may still add it - if opposite feedback is received, in which case the normal grpc server will be used for all services and we won't register with the secure. 

The first is required ( normal grpc ), the second optional.